### PR TITLE
dev-python/nuitka: bump version to 0.6.2

### PR DESCRIPTION
**Pull request by oz123**
Signed-off-by: Oz N Tiram <oz.tiram@gmail.com>
Bug: https://bugs.gentoo.org/678320
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

File: dev-python/nuitka/nuitka-0.6.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline dev-python/nuitka/nuitka-0.6.2.ebuild]: 7 Should work here.

File: dev-python/nuitka/nuitka-0.6.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+PYTHON_COMPAT=( python2_7 python3_5 python3_6)
```
Comment (juippis) [inline dev-python/nuitka/nuitka-0.6.2.ebuild]: Did/could you try with 3.7?

File: dev-python/nuitka/nuitka-0.6.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+inherit distutils-r1
+
+DESCRIPTION="Python to native compiler"
+HOMEPAGE="http://www.nuitka.net"
```
Comment (juippis) [inline dev-python/nuitka/nuitka-0.6.2.ebuild]: https:// is available, also for SRC_URI.

File: dev-python/nuitka/nuitka-0.6.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+SRC_URI="http://nuitka.net/releases/${P^}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (juippis) [inline dev-python/nuitka/nuitka-0.6.2.ebuild]: (not saying this is wrong, but distutils should put setuptools already in DEPEND or BDEPEND, depending on EAPI used.)

File: dev-python/nuitka/nuitka-0.6.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+RDEPEND="${DEPEND}
+	dev-python/appdirs[${PYTHON_USEDEP}]
+	dev-util/scons[${PYTHON_USEDEP}]
+	"
+
+S="${WORKDIR}/${P^}"
+
+pkg_postinst() {
+	elog "nuitka needs app-admin/chrpath for building"
```
Comment (juippis) [inline dev-python/nuitka/nuitka-0.6.2.ebuild]: Use optfeature here?

File: dev-python/nuitka/nuitka-0.6.2.ebuild (hunk 1 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.2.ebuild]: yes, it does work. I will change this.

File: dev-python/nuitka/nuitka-0.6.2.ebuild (hunk 2 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.2.ebuild]: It does work, but the dependency on scons blocks me here.  `dev-util/scons` is still only python 3.6.

File: dev-python/nuitka/nuitka-0.6.2.ebuild (hunk 5 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.2.ebuild]: @juippis thanks for suggesting this! I was not aware of this.

File: dev-python/nuitka/nuitka-0.6.2.ebuild (hunk 3 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.2.ebuild]: done

File: dev-python/nuitka/nuitka-0.6.2.ebuild (hunk 4 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.2.ebuild]: probably isn't wrong, but being explicit is good.

File: dev-python/nuitka/nuitka-0.6.2.ebuild (hunk 2 above)
Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.2.ebuild]: when the scons maintainer updated this, we can update nuitka. 
@mgorny, mind if I commit a version of scons with python3.7 ? 
according to https://travis-ci.org/SCons/scons/builds/499102473 it should work.

Comment (oz123) [reply dev-python/nuitka/nuitka-0.6.2.ebuild]: > LGTM. setuptools is correctly inherited in RDEPEND, too.
> 
> EDIT: Changed my mind about scons, that package looks like a special case with special care.

Yes, indeed. Scons has some patches in gentoo which need some special care.

**Comment by oz123** [comment]
@juippis can please you merge this PR as each. We agree that this should not be blocked by scons and Python-3.7

**Comment by juippis** [comment]
Sorry I can't merge it as I'm not a dev, but it should be pretty clear for anyone from proxy-maint team once they take a look. I can try to ping people in IRC.

**Pull request closed without merging**